src/lib/map-factory.js

```diff
@@ -1,9 +1,20 @@
+import ported from "./object-mapper/object-mapper";
+import original from "object-mapper";
 import Mapper from "./mapper";
 
-export default function createMapper() {
+export default function createMapper(options) {
+
+  let om = original;
+  let experimental = false;
+  if (!options) options = {};
+
+  if (options.experimental) {
+    om = ported;
+    experimental = true;
+  }
```

Comment:
It's much like matter of religion but it's more compact...

```
if (!options) options = {};
const om = (options.experimental) ? ported : original;
const experimental = (options.experimental);
```

Just a suggestion...